packages/core/src/browser/logger-frontend-contribution.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from 'inversify';
+import { CommandContribution, CommandRegistry, Command } from "../common/command";
+import { QuickOpenService, QuickOpenModel, QuickOpenItem, QuickOpenMode } from './quick-open';
+import { ILoggerServer, LoggerInfo } from '../common/logger-protocol';
+import { LogLevel, logLevelStrings, logLevelToString } from '../common/logger';
+
+/* Open a the quick open menu to choose a logger, then a log level.  */
+
```

Comment:
fyi: extra empty line between jsdoc and elements are redundant, please look at other places as well

Reply:
Done.